apps/emqx_bridge_hstreamdb/src/emqx_bridge_hstreamdb_connector.erl

```diff
@@ -243,11 +236,9 @@ do_on_start(InstId, Config) ->
             {error, {connect_failed, Error}}
     end.
 
-client_options(Config = #{url := ServerURL, ssl := SSL}) ->
-    GRPCTimeout = maps:get(<<"grpc_timeout">>, Config, ?DEFAULT_GRPC_TIMEOUT),
-    EnableSSL = maps:get(enable, SSL),
+client_options(#{url := ServerURL, ssl := SSL, grpc_timeout := GRPCTimeout}) ->
```

Comment:
fix: grpc_timeout key is atom not binary. and it has default value(10s), should alway exist.